### PR TITLE
BUG: Paths to install jars and jre should be absolute

### PR DESCRIPTION
**Pull request by fbudin69500**
Commit [1] replaced absolute install path with relative install path to
package SCIFIO correctly. However, jars and jre paths must be absolute when
using these paths in SCIFIO source code to find install paths.

[1] 1069996bd0dcc6f15a5864a0d61ccba17bb4fa66

### CODE REVIEW

**Comment by fbudin69500** [comment]
@thewtex @ctrueden

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -30,18 +30,23 @@ set( jarsDownloadDirectory jars )
 set( JAR_BUILD_TREE_LOCATION
   ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/${jarsDownloadDirectory}/
   )
-set( JAR_INSTALL_TREE_LOCATION
+set( JAR_INSTALL_LOCATION
   ${SCIFIO_INSTALL_LIBRARY_DIR}/${jarsDownloadDirectory}/
   )
-
+set( JAR_INSTALL_TREE_LOCATION
```
Comment (thewtex) [inline src/CMakeLists.txt]: Thus variable is now unused?

Comment (fbudin69500) [inline src/CMakeLists.txt]: JAR_INSTALL_TREE_LOCATION & JRE_BUILD_TREE_LOCATION are used here: https://github.com/InsightSoftwareConsortium/scifio-imageio/blob/master/src/itkSCIFIOImageIO.cxx.in#L300-L327

Is this what you are asking?

Comment (thewtex) [inline src/CMakeLists.txt]: @fbudin69500 Yes, looks good.

**Pull request merged**